client.go

```diff
@@ -40,34 +40,39 @@ type Client interface {
 // ResourceClient allows access to resource-specific clients
 type ResourceClient interface {
 	// Organization gets the OrganizationClient for the specific top-level organization, or user account.
-	// ErrNotTopLevelOrganization will be returned if the organization is not top-level
+	//
+	// ErrNotTopLevelOrganization will be returned at usage time if the organization is not top-level.
```

Comment:
Is there context for "at usage time"? Do you mean when calling this function?

Reply:
I meant this "when using the OrganizationClient", but actually, it's more explicit to just return the error directly here to not have to propagate it to more scoped sub-clients. I'll fix that